### ISSUE TITLE
Support inline spoiler tags in reviews

### ISSUE THREAD

**Issue by popcar2** [bookwyrm-social/bookwyrm#2896]
**Is your feature request related to a problem? Please describe.**
Bookwyrm doesn't support inline spoiler tags, this is a huge issue since I use them a lot in my Goodreads reviews. When I imported my Goodreads account to Bookwyrm, I was surprised that spoiler tags were ignored and much of my reviews could ruin it for other people. 

**Describe the solution you'd like**
Implement <spoiler>...</spoiler> tags in reviews

**Describe alternatives you've considered**
N/A

**Additional context**

Here's what my review looks like on Goodreads:

![image](https://github.com/bookwyrm-social/bookwyrm/assets/16920817/61499c79-45df-4b29-9c61-8e9380498505)

Here's what my review looks like on Bookwyrm after automatically importing my data from Goodreads:

![image](https://github.com/bookwyrm-social/bookwyrm/assets/16920817/89ce1baf-9551-4663-9b5d-4a66911d5b14)

For now I'm going to go ahead and manually edit my reviews to remove the spoiler section, but this is definitely something Bookwyrm should look into.

**Comment by hughrun** [comment]
@popcar2 I'm aware this doesn't resolve your issue with _inline_ spoilers but are you aware that Bookwyrm allows you to put a spoiler alert on an entire review?

**Comment by popcar2** [comment]
> @popcar2 I understand this doesn't resolve your issue with _inline_ spoilers but are you aware that Bookwyrm allows you to put a spoiler alert on an entire review?

Yes, but I rarely do that because it usually means nobody will bother opening the review
